docs/SUMMARY.md

```diff
@@ -9,6 +9,7 @@
   - [Actions](concepts/actions.md)
   - [State](concepts/state.md)
   - [Select](concepts/select.md)
+  - [State Token](concepts/token.md)
```

Comment:
Do you think that this belongs under `Concepts`?
I think that this page should be under `Advanced` and there should be a small section in the `State` page that introduces the idea and refers to this page.
What do you think?

Reply:
Okey, I will fix it

Reply:
done